app/views/welcome/index.html.erb

```diff
@@ -82,4 +98,16 @@
       });
     });
   </script>
+
+  <script type="text/javascript">
+    function showForm() {
+        var abSelect = document.getElementById("AB_select").value;
+        if (abSelect == "search_match_length") {
+            document.getElementById("host-wrapper").style.display = "none";
+        }
+        if (abSelect == "none") {
+            document.getElementById("host-wrapper").style.display = "block";
+        }
+    }
```

Comment:
This is a global method, which again is frowned upon unless it is required.

It might be worth going through options for the JS tomorrow as I don;t want that to delay the merge, however it does need some work.

Reply:
Sorted for this and the other script